### DISCUSSION TITLE
how to recompile python

### DISCUSSION THREAD

**Discussion by AhmadNaruto** [termux/termux-packages#7640] [General]
Anyone can guide me how to compile python in termux i want to try Python 3.10

**Comment by thunder-coding** [comment]
Python 3.10 has not yet reached stable release. Even though if you are too keen to compile it, here are the steps

1. Go through https://github.com/termux/termux-packages/wiki/Building-packages and https://github.com/termux/termux-packages/wiki/Creating-new-package

2. Update python version and tarball checksum in `packages/python/build.sh`

3. Fix patch errors and fix compilation of python

4. On successful compilation test the newly compiled python

5. Pull up a PR
